src/hooks/chats/chatSidebar.ts

```diff
@@ -38,9 +39,19 @@ export const useChatSidebarControls = () => {
     if (isExpanded) {
       collapseSidebar();
     } else {
+      dispatch(setVenueChatTabOpened());
       expandSidebar();
     }
-  }, [expandSidebar, collapseSidebar, isExpanded]);
+  }, [expandSidebar, collapseSidebar, dispatch, isExpanded]);
+
+  const togglePrivateChatSidebar = useCallback(() => {
+    if (isExpanded) {
+      collapseSidebar();
+    } else {
+      expandSidebar();
+      dispatch(setPrivateChatTabOpened(undefined));
```

Comment:
will that work? or you'd prefer a more obvious way with `undefined`?
```suggestion
      dispatch(setPrivateChatTabOpened());
```

Reply:
fixed in fb21560